fastlane/Fastfile

```diff
@@ -238,12 +257,71 @@ end
       project_slug: 'simplenote-macos',
       dsym_path: dSYM_PATH,
     )
-   
+  end
+
+  lane :build_and_upload_app_store do | options |
```

Comment:
This would benefit from a `desc` documentation – including documenting what are the valid `options` you can pass to this lane (same for all other lanes tbh, could be worth documenting valid options for all of them)

Reply:
:nods: I should have added a TODO about it. Was just leaving it for last.